drivers/net/ethernet/intel/ice/ice_main.c

```diff
@@ -28,7 +28,7 @@
 #include "ice_vsi_vlan_ops.h"
 #include <net/xdp_sock_drv.h>
 
-#define DRV_SUMMARY	"Intel(R) Ethernet Connection E800 Series Linux Driver"
+#define DRV_SUMMARY	"Intel(R) Ethernet Connection E800 Series Linux Driver +GTP-Uv12"
```

Comment:
This change makes sense for testing but isn't something that would typically go in a final patch.